app/presenters/publication_presenter.rb

```diff
@@ -11,7 +11,8 @@ def details
   end
 
   def documents
-    content_item["details"]["documents"].to_a.join("")
+    docs = content_item["details"]["attachments"].select { |a| a["locale"] == locale } if content_item["details"]["attachments"]
```

Comment:
Oh, this is clever, so you only display attachments with the same locale as the page. Nice!